### DISCUSSION TITLE
"Sink rate" and "Don't sink" warnings

### DISCUSSION THREAD

**Discussion by grizzbear6** [flybywiresim/aircraft#2331] [Help] [answered]
So I'm pretty sure these warnings were added with the flybywire version (correct me if I'm wrong), but I've noticed they go off on every single approach for me and I'm not even coming in at a steep angle. It keeps saying "sink rate" and "don't sink" and it just keeps going off for so long I can't even hear any of the other callouts. Is there something I'm doing wrong here or is there a way to turn them off/ remove them? Thank you!

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
did you spawn CnD? If you spawn on the runway or at a landing point, it'll yell that 24/7. If its super annoying turn off sys in the gpws part of the overhead.

**Reply by grizzbear6** [reply]
Yea I think that might be it. I’ve been spawning on the runway and in the air just about to land to practice landings; is that a bug in flybywire? I guess I’ll have to start a flight from the gates and see if I still have the problem. Also will turning off sys in the overhead turn off all callouts, or just the two I’m getting about sink rate?

**Reply by hotshotp** [reply]
yeah, so we haven't fixed this yet, but basically when you spawn on runway and approach you are in takeoff state and the game things you need to take off, therefore the don't sink

**Reply by grizzbear6** [reply]
Got it, well I’m glad you guys know about it. Thank you for your help!

**Comment by frankkopp** [comment]
Could that be related to the Flight Phases? I don't have this issue - neither when CnD nor when starting on Rwy. But I always activate Approach Phase in MCDU before doing the approach (if not already activiated automatically).
Could you try if this makes a difference?

**Reply by grizzbear6** [reply]
I'm fairly new here, so I'm not really sure what anything you said means lmao. But I think you're talking about spawning on the runway and spawning in the sky before approach. I get this issue every single time when landing, but I haven't spawned in the gates for a while, so I'm not sure if that will fix the issue yet. Just hoping that this is an issue that can be fixed.

**Reply by frankkopp** [reply]
No worries :)

Depending on how you currently do your landings you should have a point in time where you start to descent to the runway. You should be in Approach Mode then. You can activate it here:

![image](https://user-images.githubusercontent.com/16833201/101797450-5a8f8900-3b0a-11eb-8bb7-c0261571ffa4.png)

If you want to learn more watch some videos on this:
https://www.youtube.com/playlist?list=PL4XNcpjBFH8ENxr_OnrSraok-fF69KWNi

Also these description help a lot: 
http://www.a320dp.com/A320_DP/nav-flight-management/sys-14.4.1.html

**Reply by grizzbear6** [reply]
Great, next time I get into the sim I’ll have to try putting it into approach mode and see if that solves it for me. I appreciate you helping me, thank you!

**Reply by grizzbear6** [reply]
I’ll take a look at the links too. Thank you!

**Reply by MapleNAV-TCASFAULT** [reply]
Appr Mode is usually activated automatically after the aircraft passes the visible or sometimes invisible (decel) point

**Reply by grizzbear6** [reply]
Hm yea, so maybe it will already be in approach mode so it won’t help me. I guess I just can’t spawn on the runway or on approach until the issue is fixed.

**Reply by frankkopp** [reply]
Grizzbear, I'm happy to test. Can you describe exactly which airport, runway and MSFS settings (planning page before starting the flight) you used?

**Reply by grizzbear6** [reply]
I know I was flying around Manchester UK, Vancouver CA, Portland US, Seattle US, and Albuquerque US, and I was having the issue at all of those locations. I always do IFR low altitude. I’m getting into the sim right now if you need any information.

**Reply by grizzbear6** [reply]
Yup, I just did a landing where I spawned in the air and it’s still doing it. I went into 34R at Seattle Tacoma airport. At around 800 feet it started with “sink rate” and then it went to “don’t sink” after a little bit. It did it the entire time and didn’t stop until I heard the 30 feet callout.

**Reply by MapleNAV-TCASFAULT** [reply]
Yeah, because you spawned on the runway, try cold and dark.

**Reply by grizzbear6** [reply]
Is there no way to start at the gates with the aircraft running? Or is that what causes the bug?

**Reply by MapleNAV-TCASFAULT** [reply]
When you spawn "hot and light" (pretty cringe), the systems can't really configure itself properly, possibly due to the code not being fully compatible with such scenarios. For me, my adirs sometimes fails when I spawn midair. But back to your question no, you can't spawn at a gate or stand with the engines on sadly. Except to ctrl-e, but it wont setup the aircraft completely

**Reply by frankkopp** [reply]
> Yup, I just did a landing where I spawned in the air and it’s still doing it. I went into 34R at Seattle Tacoma airport. At around 800 feet it started with “sink rate” and then it went to “don’t sink” after a little bit. It did it the entire time and didn’t stop until I heard the 30 feet callout.

I just did it - spawning in air to runway 34R - active pause - setting up everything - approach mode was already set - no call out of sink rate or pull up - will do another try and will do a video

**Reply by grizzbear6** [reply]
That’s weird... I don’t understand why it’s just doing it for me. It’s not like anything is different.

**Reply by frankkopp** [reply]
So here is the video - not the best approach and I had trouble with the autothrottle - wouldn't work so used manual throttle - otherwise ILS AP approach as normal. Although ILS is a bit off to the right as you can see at the end. But the main point is no call outs of "sink rate" or "pull up". Can you try repeating it with my steps? (and of course the latest dev version of the mod :) )

[https://www.dropbox.com/t/91TCYi5gRhFNSaWZ](https://www.dropbox.com/t/91TCYi5gRhFNSaWZ)

**Reply by grizzbear6** [reply]
Thank you for taking the time to make the video! I never use autopilot on approach, but I can't imagine that is causing the problem. I'm in the sim now, so I'll try to play around and recreate what you did to see if I can pinpoint the issue. Also, the callouts kept repeating and I never heard the minimum callout for you, is that a bug too?

**Reply by MapleNAV-TCASFAULT** [reply]
did you set a MDA or DH?

**Reply by frankkopp** [reply]
I had one try without AP which also worked. The Minimum callout only comes if you have set the MDA or DH in the Approach Page.

**Reply by grizzbear6** [reply]
Oh ok, so how would I set those? In the computer somehow I'm guessing?

**Reply by frankkopp** [reply]
It's in the video - the number is 472 as per airport diagram (also in the video) for this ILS

**Reply by MapleNAV-TCASFAULT** [reply]
If you don't have charts, just set 250 in your DH, just to hear it

**Reply by grizzbear6** [reply]
And I would have to set that for every new flight, correct?

**Reply by frankkopp** [reply]
yes - every time if you want the callout. it works without it as well. There is lot to learn for airliners - especially if you want it as real as possible. Steep learning curve. Watch the videos on youtube :)

**Reply by MapleNAV-TCASFAULT** [reply]
So i did a few test flights... well landings. If you spawn mid air for approach and everything is configured, ils, appr phase page, etc. It will not go haywire. But once you either go around or slew the aircraft the "dont sink" and "sink rate" will play.

**Reply by grizzbear6** [reply]
I see on the top center screen a pink message saying "T.O. Inhibit" which I'm guessing stands for take-off inhibit. Not sure if that has something to do with my problem?

**Reply by MapleNAV-TCASFAULT** [reply]
T.O inhibit just hides and mutes unnecessary ECAM messages so it doesn't disturb your takeoff roll. It shouldn't have an effect on the issue that you're having right now. As I said earlier, the don't sink and sink rate callouts is a bug that even I encounter, where you just can't do much or anything atm.

**Reply by grizzbear6** [reply]
Hm, ok so I guess I'll just have to wait until they fix it, and then I can practice landings again. How will I know if it is fixed in an update?

**Reply by MapleNAV-TCASFAULT** [reply]
try out the same scenario 😄

**Reply by grizzbear6** [reply]
Yea I've been trying to follow his video, but to no avail. It also won't even allow me to click on the MCDU menu, so I can't completely follow it. This is an issue with the flybywire and not the stock plane, right? Hopefully they fix it and the problem just stops for me.

**Reply by MapleNAV-TCASFAULT** [reply]
The callouts were made by FlyByWire, so they're responsible for it. But this is the last thing (not literally) currently in priority. For now just start CnD, no harm in that. 🙂

**Reply by frankkopp** [reply]
I have done about a dozen landings on 34R now - spawning in air, spawning on runway - basic setup - with and without AP. I never got the "Don't sink" call out. Did not try Missed Approach though. But you can definitely train landings.

**Reply by grizzbear6** [reply]
Yea I’ll just have to start cold for now then. It takes me forever though 😭

**Reply by grizzbear6** [reply]
Yea that’s weird, I’m not sure why I’d be getting the warning and you’re not. Hopefully one of the creators/contributors find something.

**Reply by MapleNAV-TCASFAULT** [reply]
hope so, plz consider this help thread answered, unless you have further questions related to this

**Reply by MapleNAV-TCASFAULT** [reply]
Also add the answerer too. So Either hotshotp, frankkopp or me.

**Reply by frankkopp** [reply]
https://youtu.be/aqOvzNufyxw 

Nice vid showing an RNAV approach covering a lot of the setup required.
